astropy/io/fits/tests/test_header.py

```diff
@@ -715,54 +553,54 @@ def test_hierarch_create_and_update(self):
 
         header = fits.Header()
         with catch_warnings(VerifyWarning) as w:
-            header.update('HIERARCH BLAH BLAH', 'TESTA')
+            header.update({'HIERARCH BLAH BLAH': 'TESTA'})
```

Comment:
Is this necessary? I haven't looked at the code but is the removed command invalid now?